docs/limit-login-attempts.md

```diff
@@ -0,0 +1,95 @@
+# Limit Login Attempts
+
+To protect against brute force attacks, Altis has the ability to limit the number of login attempts possible both through normal login as well as using auth cookies.
+
+This feature blocks an IP address from making further attempts after a specified limit on retries is reached, making a brute-force attack difficult or impossible.
+
+- Limit for the number of retry attempts when logging in (for each IP).
+- Limit the number of attempts to log in using auth cookies in same way.
+- Informs user about remaining retries or lockout time on login page.
+- Optional logging, optional email notification.
+
+**Note:** In this document, we'll only show the `limit-login-attempts` configuration for brevity.
+
+To set the limit login attempts security settings, set values in your `composer.json` configuration under `extra.altis.modules.security.limit-login-attempts`. The default values are below, but all can be overriden.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"security": {
+					"limit-login-attempts": {
+						"allowed_retries": 4,      // Lock out after this many tries.
```

Comment:
Really wish JSON allowed for comments! I checked and the output in the docs looks fine though 👍